commons/dto/DonationDTO.ts

```diff
@@ -33,3 +33,12 @@ export interface StepFunctionInput extends DTO {
   requestPostId: string;
   additionalParams?: Record<string, any>;
 }
+
+export type AcceptedDonationDTO = DTO & {
+  donorId: string;
+  requestPostId: string;
+  acceptanceTime: string;
+  seekerId: string;
+  createdAt: string;
+  status?: string;
```

Comment:
if its optional it should extend | concat the original type

Reply:
Does [this](https://github.com/CraftsmenLtd/BloodConnect/pull/205/files#diff-a25adf68b1595e65a3ccaf232cd99cbfcd6cc47a5d4349ef1ee02c0a676652ff) change follows the structure you are talking about?

Reply:
that link isn't taking me anywhere

Reply:
This link should have taken you to the code block shown you in following image. The change is done in this commit:
https://github.com/CraftsmenLtd/BloodConnect/pull/205/commits/34f6dba54869c4fac230dd4fa30a979ca6f674c2

![image](https://github.com/user-attachments/assets/35ae4e7f-e926-43a6-a9d6-2087145ea215)